nnvm/python/nnvm/testing/check_computation.py

```diff
@@ -441,6 +443,23 @@ def check_function(symbol, forward=None, backward=None, grad_input_vars=None,
             debug_stage = "running"
             nnvm_res = main_function(**np_inputs)
 
+            try:
+                logging.debug("checking to_relay conversion")
```

Comment:
Will this print every time this code is executed?

Reply:
I think it depends on the logging configuration, but, with our CI, this message will be printed only when an error occurs.

Reply:
ok, seems ok